src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
Can you please move the comment on top of `#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"` again and add a comment like `// Snapdragon: use J15 (next to USB)` above `#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"`?